plugin/evm/limit_order.go

```diff
@@ -462,6 +462,7 @@ func executeFuncAndRecoverPanic(fn func(), panicMessage string, panicCounter met
 
 			log.Error(panicMessage, "errorMessage", errorMessage, "stack_trace", string(debug.Stack()))
 			panicCounter.Inc(1)
+			orderbook.AllPanicsCounter.Inc(1)
```

Comment:
do we need 2 counters?

Reply:
there was already a source specific panic counter(specified by the caller), and i added another one for all kinds of panics, just for monitoring

we don't really need 2 separate, but feel it's good to have